site/.vitepress/theme/tokens.css

```
@@ -1,167 +1,179 @@
+  --sh-dim:        #666;
+  --sh-cmd:        #6d9300;
+  --sh-arg:        #444;
 
-  /* ===== Depth — one shadow, never stacked, never a coloured glow ===== */
-  --shadow-whisper: 0 1px 2px oklch(0% 0 0 / 0.07);
+  /* ===== Depth — Hub のボタン影 ===== */
+  --shadow-btn:    0 8px 20px -5px rgba(0, 0, 0, 0.2);
+  --shadow-accent: 0 8px 20px -5px rgba(134, 179, 0, 0.4);
 
   /* ===== Type ===== */
   --font-body: Nunito, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Hiragino Sans', 'BIZ UDGothic', sans-serif;
```

Stylelint (17.14.1)

[error] Expected "Nunito" to be "nunito" (value-keyword-case)

(value-keyword-case)